.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,17 @@
 ## Scope of this PR
 
-<!-- REQUIRED We set [labels](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#devdocs) based on the scope of changes with a PR. -->
+<!-- REQUIRED
+1. Leave the label that best describes the scope of your PR visible.
```

Comment:
```suggestion
1. Choose one of the following labels that best describes the scope of your PR and do not delete it.
```

Reply:
TBH this seems longer and less clear. I like the specificity of it, but we're going for a quick understanding of what's needed.

Reply:
I would rather explicitly tell them to also delete the others rather than imply it.